client/src/views/Home.tsx

```diff
@@ -36,10 +36,9 @@ export const Home: React.FC = (propps) => {
       <Container my="md">
         <Grid>
           <Grid.Col xs={12}>{<Info />}</Grid.Col>
-          {
-            elections.map(e => 
-              <Grid.Col xs={4}>{<ElectionInfo election={e}/>}</Grid.Col>)
-          }
+          {elections.map((e) => 
+            <Grid.Col xs={4}>{<DisplayElectionInfo election={e} modalContent={Voting}/>}</Grid.Col>
```

Comment:
I prefer it when you do like this and use components called `Grid` and so on. In some places you use classes called `FlexContainer`. Why is there not a component for this as well? They usually exist in component libraries.